videoencoder.cpp

```diff
@@ -136,18 +128,18 @@ rgb_to_yuv (Mat * image, th_ycbcr_buffer ycbcr)
   for (y = 0; y < h; y++)
     {
       for (x = 0; x < w; x++)
-	{
-	  unsigned char b = image->data[image->channels () * (image->cols * y + x) + 0];
-	  unsigned char g = image->data[image->channels () * (image->cols * y + x) + 1];
-	  unsigned char r = image->data[image->channels () * (image->cols * y + x) + 2];
-
-	  yuv_y[x + y * yuv_w] =
-	    clamp ((65481 * r + 128553 * g + 24966 * b + 4207500) / 255000);
-	  yuv_u[x + y * yuv_w] =
-	    clamp ((-33488 * r - 65744 * g + 99232 * b + 29032005) / 225930);
-	  yuv_v[x + y * yuv_w] =
-	    clamp ((157024 * r - 131488 * g - 25536 * b + 45940035) / 357510);
-	}
+  {
+    unsigned char b = image->data[image->channels () * (image->cols * y + x) + 0];
```

Comment:
@coolo did a re-check against the version in master, the file has a mix between tabs and spaces (and it needs some beautify too)... These are spaces actually